### ISSUE TITLE
import from rendered Remark.js presentations

### ISSUE THREAD

**Issue by blahah** [slidewinder/slidewinder#29]

**Comment by TransGirlCodes** [comment]
Would you like me to do something for this, it's marked help wanted and not urgent, so nothing we do should clash. I see this as working through the framework extensions mechanism. So frameworks currently (last time I checked!) define some pre-processors, a deck template helpers, and a html template. Well we just have these frameworks also specify an importer method, and then give a command line utility do be able to do this. 

EDIT - Actually, seeing other import issues - it's probably better to introduce a class importer, which like framework, can support various formats.

**Comment by blahah** [comment]
I agree with the second idea - if you'd like to start on the generalised importing that would be great. And of the importers, powerpoint is the most urgent I think -> #26

**Comment by TransGirlCodes** [comment]
I agree, Power-Point is most urgent. Ok! I'll make this my evening's coding activity!
